hudi-common/src/main/java/org/apache/hudi/internal/schema/convert/AvroInternalSchemaConverter.java

```diff
@@ -91,6 +91,11 @@ public static InternalSchema convert(Schema schema) {
     return new InternalSchema(fields);
   }
 
+  /** Convert an avro schema into internalSchema with given versionId. */
+  public static InternalSchema convertToEmpty(Schema schema) {
+    return new InternalSchema(InternalSchema.EMPTY_SCHEMA_VERSION_ID, schema);
```

Comment:
This is also confusing, an internal schema with 'empty' version id but still got avro schema internal, please clarify it.

Reply:
empty schema means schema evolution disabled or appropriate internal schema not found

Reply:
So a avro schema with `empty` version id represents which case ?

Reply:
avro schema with empty version id represents disabled schema evolution

Reply:
getEmptyInternalSchema  will give a emptyInternalSchema which indicate schema evolution disabled.